libs/zzz/util/src/level.ts

```
@@ -39,7 +29,7 @@ export function validateLevelAsc(
   level: number,
   ascension: AscensionKey
 ): { level: number; ascension: AscensionKey } {
-  if (typeof level !== 'number' || level < 1 || level > 90) level = 1
+  if (typeof level !== 'number' || level < 1 || level > 60) level = 1
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)